charts/synthetics-private-location/CHANGELOG.md

```diff
@@ -1,12 +1,16 @@
 # Datadog changelog
 
+## 0.15.8
+
+* Fix commonLabels duplicated in Deployment.
+
 ## 0.15.7
 
 * Update private location image version to `1.31.1`.
 
 ## 0.15.6
 
-* Update private location image version to `1.31.0`.
+* Fix commonLabels duplicated in Deployment.
```

Comment:
You've overwritten a previous version Changelog
```suggestion
* Update private location image version to `1.31.0`.
```

Reply:
indeed, sorry.